erpnext/selling/page/point_of_sale/point_of_sale.js

```diff
@@ -922,7 +939,25 @@ class POSCart {
 		this.$qty_total.find('.quantity-total').text(total_item_qty);
 		this.frm.set_value("pos_total_qty",total_item_qty);
 	}
-
+	make_order_type_options()
+	{
```

Comment:
Pull this to the previous line, it'll be consistent with the rest of the functions